test/server/lib/db/query.fixtures.js

```diff
@@ -80,5 +80,36 @@ WHERE date_time >= $(from) AND date_time < $(to) AND groups::TEXT ILIKE '%' || $
     gorupOnlySQL: `SELECT count(*)::INTEGER
 FROM lev_audit
 WHERE groups::TEXT ILIKE '%' || $(group) || '%'`
+  },
+  hourlyUsage: {
+    // NOTE: the following query would take around 30 SECONDS to complete, so will 502
+    // It may be possible to re-include this kind full data request once the the
+    // underlying infrastructure can handle it, and we deem it useful!
+//     noParameterSQL: `SELECT COUNT(*)::INTEGER, weekend::INTEGER, hour
+// FROM (
+//   SELECT TO_CHAR(date_time AT TIME ZONE 'europe/london', 'HH24')::INTEGER AS hour,
+//     TO_CHAR(date_time AT TIME ZONE 'europe/london', 'DAY') LIKE 'S%' AS weekend
+//   FROM lev_audit
+//   WHERE groups::TEXT NOT ILIKE '%monitor%' AND client = 'lev-web'
+// ) AS counts
+// GROUP BY weekend, hour`,
```

Comment:
Is this still true? If so, how will we be able to run this in production?

Reply:
Well if no start date is provided, the report assumes the start of the month, so there can't ever be a situation where the query would run without that filter. This and the test for an error being thrown, is more to document that we wouldn't want to try and fetch the full data.

Reply:
Okay.